### ISSUE TITLE
Find partial citations for decisions that don't yet have full citations, e.g. 22 U.S. ____ (2002)

### ISSUE THREAD

**Issue by mlissner** [freelawproject/eyecite#30]
Back in 2014, in https://github.com/freelawproject/courtlistener/issues/1601, @brianwc lamented that we don't handle partial citations very well. His example was:

> The Supreme Court often cites its recent cases that don't yet have a full U.S. citation like so:
> 
> Bullcoming v. New Mexico, 564 U. S. ___ (2011).
> 
> We have Bullcoming v. New Mexico, know it is a SCOTUS case, and know it was filed in 2011. 

Alas, we never got the citator working on these. I'm not sure it's worth the effort to do so now, but I thought I'd file an issue so we could track it as a gap in functionality.

**Comment by mattdahl** [comment]
Relinking my old PR on this: https://github.com/freelawproject/courtlistener/pull/1226

On the parsing side, I think this may be as simple as allowing "__" to be a valid page number here: https://github.com/freelawproject/eyecite/blob/master/eyecite/utils.py#L31

**Comment by mlissner** [comment]
Yeah, I think we should start by focusing on the parsing side first. On the matching side, we have competing proposals of either:

1. Building up a list of all possible volumes for all reporters; or
2. Just doing lookups based on ± 1-2 years from the date of the opinion.

I actually think number two is more practical and less fragile. I'd say if we revisit this once the parser is done, that we take that approach.

**Comment by devlux76** [comment]
I'm volunteering to pitch in on a solution to this if you'll have me.
I'm a 2YL, just finished up my "electronic legal research" and "legal analysis & writing a.k.a how to cite to bluebook" courses. I got decent grades. I'm also a moderately competent coder. I've got some spare time on my hands before it's back to law school.

I have some ideas I'd like to lay down. Just don't want to step on anyone's toes.

**Comment by mlissner** [comment]
Go, go, go, @devlux76. We'd love the help. I think I highlighted one other issue that would be a good first-bug, but this would be a good one too, I think.

**Comment by flooie** [comment]
@mlissner would you like to revive this?

**Comment by mlissner** [comment]
T shirt size estimate?
